src/category/category.test.ts

```diff
@@ -0,0 +1,64 @@
+import { CategoryFactory, CategoryFactoryParams, CategoryItemFactory, CategoryLevelFactory } from './factory'
+import { LanguageCode } from 'src/language'
+
+test('ExpectedCategory === Category', () => {
+  const categoryParams: CategoryFactoryParams = {
+    props: {
+      name: 'name',
+    },
+  }
+
+  const category = CategoryFactory.createInstance(categoryParams)
+
+  expect(category).toHaveProperty('props')
+  expect(category.props).toHaveProperty('name')
+  expect(category.props.name).toBe('name')
```

Comment:
USE MOCK VALUES TO VERIFY VALUES!

Reply:
It was a bad merge, i think that now it is solved